docs/process-typed.md

```diff
@@ -5,6 +5,14 @@
 :::{versionadded} 25.10.0
 :::
 
+:::{warning}
+This is a preview feature – the syntax and behavior may change in future releases.
+:::
+
+:::{note}
+This feature requires the `nextflow.preview.types` feature flag to be enabled in every script that uses typed processes.
+:::
```

Comment:
I think we originally did this but @christopher-hakkaart preferred to give step-by-step instructions after the example because it is more comprehensive